config/x86/kernel-config

```diff
@@ -982,41 +1002,36 @@ CONFIG_TRANSPARENT_HUGEPAGE_ALWAYS=y
 # CONFIG_TRANSPARENT_HUGEPAGE_MADVISE is not set
 CONFIG_ARCH_WANTS_THP_SWAP=y
 CONFIG_THP_SWAP=y
-CONFIG_CLEANCACHE=y
+# CONFIG_CLEANCACHE is not set
```

Comment:
Not sure if we really want disable this one https://github.com/torvalds/linux/blob/master/mm/Kconfig#L448-L468